src/main/resources/db/migration/V1_0__create_tables.sql

```diff
@@ -16,9 +16,9 @@ CREATE TABLE visit
 CREATE TABLE visit_visitor
 (
     visit_id        integer        NOT NULL,
-    contact_id      integer        NOT NULL,
+    nomis_Person_id      integer        NOT NULL,
     lead_visitor    BOOLEAN,
-    PRIMARY KEY (visit_id, contact_id)
+    PRIMARY KEY (visit_id, nomis_person_id)
 );
```

Comment:
I'm dropping the tables - so fine to directly update this flyway script

Reply:
LGTM